pkg/virt-operator/install-strategy/patches.go

```diff
@@ -147,7 +147,7 @@ func (c *Customizer) GetPatchesForResource(resourceType, name string) []v1.Custo
 	patches := make([]v1.CustomizeComponentsPatch, 0)
 
 	for _, p := range allPatches {
-		if strings.EqualFold(p.ResourceType, resourceType) && strings.EqualFold(p.ResourceName, name) {
+		if (p.ResourceType == "" || strings.EqualFold(p.ResourceType, resourceType)) && (p.ResourceName == "" || strings.EqualFold(p.ResourceName, name)) {
```

Comment:
I would propose to use `"*"` since `""` is default value of `string`.

Reply:
The behavior when the value is unspecified is what didn't make sense to me. If I don't specify `ResourceName`, I expected to match any. The current behavior is that an unspecified `ResourceName` or `ResourceValue` would never match anything (as these values are never empty).

If you want to go the wildcard route then I'd argue that these fields should be mandatory (they are not currently as implied by `omitempty`).

Reply:
I think we should go with the wildcard asterisk, like k8s does in the webhook. That we allow right now empty values is the real issu. We should just fail on validation there.

Reply:
The problem is that we don't have validation(webhook, nor crds validation). I agree with dropping `omitempty` as I believe it has no meaning. If it isn`t too much to ask, could you add validation webhook?

Reply:
> The problem is that we don't have validation(webhook, nor crds validation). I agree with dropping omitempty as I believe it has no meaning. If it isn`t too much to ask, could you add validation webhook?

I would be fine to just address the `*` use-case in this PR. I think adding the webhook may be too much. @muff1nman except if you feel comfortable with adding the webhook of course.